decode_test.go

```diff
@@ -16,6 +16,8 @@ age = 250
 andrew = "gallant"
 kait = "brady"
 now = 1987-07-05T05:45:00Z
+nowCanada = 2017-06-22T02:14:36-06:00
+nowChina = 2017-06-22T16:15:21+08:00
```

Comment:
Hah. Sorry, I didn't see that this was already merged before doing a review. 🤦‍♂️